care/facility/api/viewsets/facility.py

```diff
@@ -204,3 +204,19 @@ def get_serializer_context(self):
         context = super().get_serializer_context()
         context["facility"] = facility
         return context
+
+
+class FacilityHubsViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    queryset = FacilityHubSpoke.objects.all().select_related("spoke", "hub")
+    serializer_class = FacilitySpokeSerializer
+    permission_classes = (IsAuthenticated, DRYPermissions)
```

Comment:
```suggestion
    permission_classes = (IsAuthenticated,)
```
drypermissions is not really doing anything here, we can remove it.